.readthedocs.yml

```diff
@@ -7,7 +7,7 @@ build:
   jobs:
     pre_build:
       # Set conf vars and  update the Sphinx configuration (conf.py)
-      - idaesx conf --execute cache --timeout 600 --sphinx --show
```

Comment:
@dangunter could you sign off on this change? As far as I can tell it seems to be doing the right thing now, but maybe there's some subtlety I'm missing.